routes/web.php

```diff
@@ -13,6 +14,8 @@
 |
 */
 
+Route::resource('users', UserController::class);
```

Comment:
Since you are using a resource route controller here, but not all the methods are defined in the controller, what happens when one calls a route with an un-implemented controller function? 

This should be replaced with a Route::<method>  for the implemented routes only. I believe laravel configuration also offers the ability to include/exclude routes using `Route::resource` but you'll need to double check that.

Reply:
Ah yes, here's the docs: https://laravel.com/docs/8.x/controllers#restful-partial-resource-routes

Reply:
you can use `Route::resource(...)->only('store', 'update', ...)` to only create routes for specific restful methods on a controller.

Reply:
Updated the route